packages/graphql/src/schema/resolvers/update.ts

```diff
@@ -48,21 +49,25 @@ export default function updateResolver({ node }: { node: Node }) {
             ...(nodeProjection ? { [nodeKey]: executeResult.records.map((x) => x.this) } : {}),
         };
     }
+    const relationFields: Record<string, string> = node.relationFields.length
+        ? {
+              connect: `${node.name}ConnectInput`,
+              disconnect: `${node.name}DisconnectInput`,
+              create: `${node.name}RelationInput`,
+              delete: `${node.name}DeleteInput`,
+          }
+        : {};
 
+    if (schemaComposer.has(`${node.name}ConnectOrCreateInput`)) {
+        relationFields.connectOrCreate = `${node.name}ConnectOrCreateInput`;
+    }
```

Comment:
In what scenario would just checking for `node.relationFields.length` not be sufficient here?

Reply:
connectOrCreate only exists for relations with `@unique` fields.

On a related note, I think it is a bit more safe to check for the existence of a type rather than duplicating the checks used for creating the type, if in a couple of months, `connectOrCreate` require other conditions, we won't need to change that in 2 separate places, as the type creation holds the single source of truth of whether the type exists or not